project/src/TerrainSprites.gd

```diff
@@ -0,0 +1,55 @@
+tool
+extends Node2D
+
+export var terrain_textures: Array
+export var NUM_TERRAIN_SPRITES: int = 15
+
+const SCREEN_SPACE = Vector2(512, 448)
+const HALF_SCREEN_SPACE = Vector2(SCREEN_SPACE.x / 2, SCREEN_SPACE.y / 2)
+const QUART_SCREEN_SPACE = Vector2(SCREEN_SPACE.x / 4, SCREEN_SPACE.y / 4)
+const CHUNK_SIZE = Vector2(1024, 896)
```

Comment:
Can this be derived from `SCREEN_SPACE`? Or is it decoupled?

Reply:
It is decoupled.